sdk/test/metrics/observer_result_test.cc

```diff
@@ -0,0 +1,36 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+#ifndef ENABLE_METRICS_PREVIEW
+#  include "opentelemetry/sdk/metrics/observer_result.h"
+#  include "opentelemetry/sdk/metrics/view/attributes_processor.h"
+
+#  include <gtest/gtest.h>
+
+using namespace opentelemetry::sdk::metrics;
+TEST(ObserverResult, BasicTests)
+{
+  const AttributesProcessor *attributes_processor = new DefaultAttributesProcessor();
```

Comment:
I couldn't find where `attributes_processor` is freed.

Reply:
Good catch. Cleaned up now after tests are completed.